CONTRIBUTING.md

```diff
@@ -130,32 +128,37 @@ jobs:
 
 ### How to contribute code to packit
 
-1. Create a fork of the `packit` repository.
+1. Create a fork of this repository.
 2. Create a new branch just for the bug/feature you are working on.
 3. Once you have completed your work, create a Pull Request, ensuring that it meets the requirements listed below.
 
-### Requirements for Pull Requests
+### Requirements for Pull Requests (PR)
 
-- Please create Pull Requests against the `master` branch.
-- Please make sure that your code complies with [PEP8](https://www.python.org/dev/peps/pep-0008/).
-- One line should not contain more than 100 characters.
-- Make sure that new code is covered by a test case (new or existing one).
-- We don't like [spaghetti code](https://en.wikipedia.org/wiki/Spaghetti_code).
-- The tests have to pass.
+- Use `pre-commit` (see below).
+- Use common sense when creating commits, not too big, not too small. You can also squash them at the end of review.
```

Comment:
What about mentioning something like this:
https://chris.beams.io/posts/git-commit/